util/ranger/ranger_test.go

```diff
@@ -346,56 +346,56 @@ func (s *testRangerSuite) TestIndexRange(c *C) {
 			exprStr:     `a LIKE 'abc%'`,
 			accessConds: `[like(test.t.a, abc%, 92)]`,
 			filterConds: "[]",
-			resultStr:   "[[abc <nil>,abd <nil>)]",
+			resultStr:   "[[\"abc\" NULL,\"abd\" NULL)]",
 		},
 		{
 			indexPos:    0,
 			exprStr:     "a LIKE 'abc_'",
 			accessConds: "[like(test.t.a, abc_, 92)]",
 			filterConds: "[like(test.t.a, abc_, 92)]",
-			resultStr:   "[(abc +inf,abd <nil>)]",
+			resultStr:   "[(\"abc\" +inf,\"abd\" NULL)]",
 		},
 		{
 			indexPos:    0,
 			exprStr:     "a LIKE 'abc'",
 			accessConds: "[like(test.t.a, abc, 92)]",
 			filterConds: "[]",
-			resultStr:   "[[abc,abc]]",
+			resultStr:   "[[\"abc\",\"abc\"]]",
 		},
 		{
 			indexPos:    0,
 			exprStr:     `a LIKE "ab\_c"`,
 			accessConds: "[like(test.t.a, ab\\_c, 92)]",
 			filterConds: "[]",
-			resultStr:   "[[ab_c,ab_c]]",
+			resultStr:   "[[\"ab_c\",\"ab_c\"]]",
 		},
 		{
 			indexPos:    0,
 			exprStr:     `a LIKE '%'`,
 			accessConds: "[]",
 			filterConds: `[like(test.t.a, %, 92)]`,
-			resultStr:   "[[<nil>,+inf]]",
+			resultStr:   "[[NULL,+inf]]",
 		},
 		{
 			indexPos:    0,
 			exprStr:     `a LIKE '\%a'`,
 			accessConds: "[like(test.t.a, \\%a, 92)]",
 			filterConds: "[]",
-			resultStr:   `[[%a,%a]]`,
+			resultStr:   `[["%a","%a"]]`,
 		},
 		{
 			indexPos:    0,
 			exprStr:     `a LIKE "\\"`,
 			accessConds: "[like(test.t.a, \\, 92)]",
 			filterConds: "[]",
-			resultStr:   `[[\,\]]`,
+			resultStr:   "[[\"\\\",\"\\\"]]",
 		},
 		{
 			indexPos:    0,
 			exprStr:     `a LIKE "\\\\a%"`,
 			accessConds: `[like(test.t.a, \\a%, 92)]`,
 			filterConds: "[]",
-			resultStr:   `[[\a <nil>,\b <nil>)]`,
+			resultStr:   "[[\"\\a\" NULL,\"\\b\" NULL)]",
```

Comment:
Why change the quotation?

Reply:
this is due to the modification of `formatDatum` in **util/ranger/types.go**

Reply:
i mean use `, we don't need escape characters

Reply:
Oh, the result was directly copied from the test output😂

Reply:
The usage of ` and " is not uniformed now, there are many tests use " actually.